build/gke-test-cluster/cluster.yml.jinja

```diff
@@ -34,6 +34,21 @@ resources:
               - https://www.googleapis.com/auth/devstorage.read_only
               - https://www.googleapis.com/auth/logging.write
               - https://www.googleapis.com/auth/monitoring
+        - name: "agones-metrics"
+          initialNodeCount: 1
+          config:
+            machineType: n1-standard-4
```

Comment:
Should this be 4 cpu?
(Should this be in the same nodepool as the controller?)

Reply:
I think those will be different shapes of VMs, so probably not (prometheus may get really large, while the controller likely won't).